### PR TITLE
10-additional ZaRR UI enhancements

### PR DESCRIPTION
**Pull request by A-Souhei**
## Description

- changes object fit for organization logo image in organization_item
- dashboard tab for organization now have same margin top like others

Relates https://github.com/fjelltopp/ckanext-zarr/pull/11

image
image

## Checklist

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request).
You may not need to check all boxes.

- [x] The Jira ticket for this issue has been updated to "Ready to Review" or equivalent.
- [x] I have developed these changes in discussion with the appropriate project manager.
- [ ] My code follows the general Fjelltopp documentation (see Confluence).
- [ ] I have made corresponding changes to the Fjelltopp documentation (see Confluence).
- [ ] I have rebased this branch with master.
- [ ] New dependency changes have been committed.
- [ ] I have added automated tests that prove my fix is effective or that my feature works.
- [x] New and existing tests pass locally with my changes.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own code.
- [x] I have assigned at least one reviewer.
- [ ] I have assigned at least one label to this PR: "patch", "minor", "major".

### CODE REVIEW

File: ckanext/fjelltopp_theme/assets/css/fjelltopp-theme.scss [hunk 1]
```diff
@@ -26,7 +26,7 @@ body {
     border-bottom: none;
 }
 
-.main, .homepage .hero {
+.main, .homepage .hero, .dashboard-organisation-main {
```
Comment (ChasNelson1990) [inline ckanext/fjelltopp_theme/assets/css/fjelltopp-theme.scss]: Why does organisation need to be treated differently to the other tabs? Surely there should be one rule that styles all four tabs?

File: ckanext/fjelltopp_theme/assets/css/fjelltopp-theme.scss [hunk 2]
```diff
@@ -1611,11 +1611,12 @@ a.thumbnail.active {
 
 .media-grid .media-image {
     width: auto;
-    aspect-ratio: 9 / 4;
+    aspect-ratio: 16/ 9;
```
Comment (ChasNelson1990) [inline ckanext/fjelltopp_theme/assets/css/fjelltopp-theme.scss]: Why change AR?

File: ckanext/fjelltopp_theme/assets/css/fjelltopp-theme.scss [hunk 3]
```diff
@@ -1611,11 +1611,12 @@ a.thumbnail.active {
...
     height: auto;
     max-width: 100%;
     display: block;
-    object-fit: cover;
+    object-fit: contain;
+    background-color: config.$light;
```
Comment (ChasNelson1990) [inline ckanext/fjelltopp_theme/assets/css/fjelltopp-theme.scss:1619]: :+1:

File: ckanext/fjelltopp_theme/templates/snippets/subscribe_to_updates.html [hunk 4]
```diff
@@ -2,8 +2,9 @@
 <div class="subscribe">
     <div class="container">
         <div class="highlight-box">
+            {% block contents %}
             <h2>{{ _('Subscribe to updates') }}</h2>
-            <div class="subtext">{{ _('Create an account to receive notifications by email or SMS whenever new documents are posted.') }}</div>
+            <div class="subtext">{% block subtext %}{{ _('Create an account to receive notifications by email or SMS whenever new documents are posted.') }}{% endblock %}</div>
```
Comment (ChasNelson1990) [inline ckanext/fjelltopp_theme/templates/snippets/subscribe_to_updates.html]: Not sure why you've made these changes? The plan was just to remove the words "or SMS", right?

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]

File: ckanext/fjelltopp_theme/templates/snippets/subscribe_to_updates.html (hunk 4 above)
Comment (A-Souhei) [reply ckanext/fjelltopp_theme/templates/snippets/subscribe_to_updates.html]: @ChasNelson1990 To extend to subcribe_to_updates.html I have added the blocks. Should I just copy/paste the original and change the text?

File: ckanext/fjelltopp_theme/assets/css/fjelltopp-theme.scss (hunk 2 above)
Comment (A-Souhei) [reply ckanext/fjelltopp_theme/assets/css/fjelltopp-theme.scss]: Put back to previous AR, logo is a bit smaller though.

File: ckanext/fjelltopp_theme/templates/snippets/subscribe_to_updates.html (hunk 4 above)
Comment (ChasNelson1990) [reply ckanext/fjelltopp_theme/templates/snippets/subscribe_to_updates.html]: Ah, I see you are changing this in ckanext-zarr.  I think I would change this text here as we don't currently offer SMS updates on most of our platforms.

Comment (A-Souhei) [reply ckanext/fjelltopp_theme/templates/snippets/subscribe_to_updates.html]: Alright, I removed the 'or SMS' here.

**Review by ChasNelson1990 — APPROVED** [review]
LGTM

**Pull request merged**